test/functional/tests/cli/test_cas_memory_usage.py

```diff
@@ -0,0 +1,136 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas.cas_module import CasModule
+from core.test_run import TestRun
+from test_utils.size import Unit, Size
+from test_utils.os_utils import (allocate_memory,
+                                 defaultize_memory_affecting_functions,
+                                 disable_memory_affecting_functions,
+                                 drop_caches,
+                                 get_free_memory,
+                                 is_kernel_module_loaded,
+                                 load_kernel_module,
+                                 unload_kernel_module,
+                                 unmount_ramfs,
+                                 get_dut_cpu_number)
+
+
+@pytest.mark.os_dependent
+def test_cas_module_memory_usage():
+    """
+        title: Validate OpenCAS kernel module memory usage.
+        description: |
+          Check that OpenCAS kernel module memory usage is in acceptable limits.
+        pass_criteria: |
+          - Loaded OpenCAS kernel module should not consume more than 110% of total memory
+          calculated by requirement formula:
+          7.5 MiB * active CPUs number
+    """
+    with TestRun.step("Disable caching and memory over-committing."):
+        disable_memory_affecting_functions()
+        drop_caches()
+
+    with TestRun.step("Measure memory usage without OpenCAS module."):
+        if is_kernel_module_loaded(CasModule.cache.value):
+            unload_kernel_module(CasModule.cache.value)
+            unload_kernel_module(CasModule.disk.value)
+        available_mem_before_cas = get_free_memory()
+
+    with TestRun.step("Load OpenCAS module"):
+        output = load_kernel_module(CasModule.cache.value)
+        if output.exit_code != 0:
+            TestRun.fail("Cannot load OpenCAS module!")
+
+    with TestRun.step("Measure memory usage with OpenCAS module."):
+        available_mem_with_cas = get_free_memory()
+        memory_used_by_cas = available_mem_before_cas - available_mem_with_cas
+        TestRun.LOGGER.info(
+            f"OpenCAS module uses {memory_used_by_cas.get_value(Unit.MiB):.2f} MiB of DRAM."
+        )
+
+    with TestRun.step("Validate amount of memory used by OpenCAS module"):
+        expected_memory_used = Size(8.0 * get_dut_cpu_number(), Unit.MiB)
```

Comment:
Why are you using 8.0MiB in calculation, when there is 7.5MiB mentioned in test description? Please correct values.